lib/controller.rb

```diff
@@ -83,15 +83,16 @@ class SlowFood < Sinatra::Base
     redirect '/auth/login'
   end
 
+  get '/menu' do
+    @menu = Menu.first
+    @dishes = Dish.all
```

Comment:
Instead of searching for all dishes, we only want to see the dishes that are related to that menu. Remember `@menu.dishes`? Try it.